lib/tasks/gettext.rake

```diff
@@ -1,21 +1,23 @@
 # -*- encoding : utf-8 -*-
 namespace :gettext do
-  def msgmerge
-    Rails.application.config.gettext_i18n_rails.msgmerge ||
```

Comment:
I attempted to set this config in the initializers but not all these rake tasks actually load the full Rails application so these options need to be defined here anyway.

Reply:
👍   – squash into commit message